src/lib/components/drs/formComponents/DrsObjectRelatives.js

```diff
@@ -0,0 +1,90 @@
+                                <FormControl fullWidth>
+                                    <TextField
+                                        fullWidth
+                                        variant='outlined'
+                                        id={`ID_${props.relationship}${index}`} 
+                                        label='Id' name={relative.id} value={relative.id} margin='normal' type='text' 
+                                        onChange={e => {
+                                            props.setRelativeId(index, e.target.value);
+                                            props.unsetRelativeValidity(index);
+                                        }}
+                                        InputProps={
+                                            {
+                                                readOnly: props.readOnly,
+                                                endAdornment: 
+                                                    <InputAdornment position='end'>
+                                                        <VerifyIdButton 
+                                                            activeDrsObject={props.activeDrsObject}
+                                                            relative={relative}
+                                                            setRelativeName={(name) => props.setRelativeName(index, name)}
+                                                            setRelativeValid={() => props.setRelativeValid(index)}
+                                                            setRelativeInvalid={() => props.setRelativeInvalid(index)}
+                                                            retrieveDrsObject={props.retrieveDrsObject}
+                                                            readOnly={props.readOnly}
+                                                        />
+                                                    </InputAdornment>
+                                            }
+                                        }
+                                    />
+                                </FormControl>
+                            </Grid>
+                            <Grid item xs>
+                                <FormControl fullWidth>
+                                    <TextField variant='outlined' fullWidth id={`Name_${props.relationship}${index}`} 
+                                    label='Name' margin='normal' name={relative.name} type='text' 
+                                    value={relative.name} InputProps={{readOnly: true}}/>                            
+                                </FormControl>
+                            </Grid>
+
+                            <RemoveItemButton
+                                display={!props.readOnly}
+                                objectName={props.objectName}
+                                handleClick={() => props.removeRelative(index)}
+                            />
```

Comment:
There is no longer a "Details" button displayed on the SHOW page (i.e. when readOnly == true). Therefore, from the SHOW page the user cannot navigate directly to the related DRS Objects.